tests/unit_tests/fake_pruned_blockchain.cpp

```diff
@@ -34,6 +34,7 @@
 //local headers
 #include "common/container_helpers.h"
 #include "ringct/rctOps.h"
+#include "ringct/rctSigs.h"
```

Comment:
Why this header?

Reply:
Remnant of the prior round of changes, will remove